packages/plugin-expo-eas-sourcemaps/lib/eas-build-on-success.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+const { access } = require('fs').promises
+const { reactNative } = require('@bugsnag/source-maps')
+const { exit } = require('process')
+
+const PROJECT_ROOT = process.cwd()
+
+if (process.env.EAS_BUILD_PLATFORM !== 'android') {
+  console.log('Skipping Android source map upload: iOS build detected')
```

Comment:
Could this also be web?

```suggestion
  console.log('Skipping Android source map upload: Android build not detected')
```